generator/generator_tests/metadata_parser_test.cpp

```diff
@@ -212,6 +212,8 @@ UNIT_TEST(Metadata_ValidateAndFormat_wikimedia_commons)
   p(kWikiKey, "Category:Bosphorus");
   TEST_EQUAL(md.Get(Metadata::FMD_WIKIMEDIA_COMMONS), "Category:Bosphorus", ());
 
+  TEST_EQUAL(md.ToWikimediaCommonsURL("Category:What If? (Bonn)"), "https://commons.wikimedia.org/wiki/Category:What If%3F (Bonn)", ());
```

Comment:
Are URLs with spaces opened properly on iOS and Android?

Reply:
Redirect means a slower server response. Does it make sense to replace spaces with underscores too? What other symbols should be escaped? Is there a full list of all category values to test?